ext/date/tests/bug62852_var2.phpt

```diff
@@ -1,31 +0,0 @@
---TEST--
-Bug #62852 (Unserialize invalid DateTime causes crash), variation 2
---INI--
-date.timezone=GMT
---FILE--
-<?php
-$s2 = 'O:3:"Foo":3:{s:4:"date";s:20:"10007-06-07 03:51:49";s:13:"timezone_type";i:3;s:8:"timezone";s:3:"UTC";}';
```

Comment:
This test should not be removed, but instead, a different *wrong* date string should be used that is unparsable.